### PR TITLE
Add rke v0.1.10

### PR DESCRIPTION
**Pull request by johnsockwell**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
[Related Pull Request](https://github.com/Homebrew/homebrew-cask/pull/52864)

### CODE REVIEW

File: Formula/rke.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Rke < Formula
+  desc "Rancher Kubernetes Engine, a Kubernetes installer that works everywhere"
+  homepage "https://rancher.com/docs/rke/v0.1.x/en/"
+  url "https://github.com/rancher/rke.git",
+    :tag => "v0.1.9"
```
Comment (fxcoudert) [inline Formula/rke.rb]: Needs a `:revision` line

File: Formula/rke.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/rancher/rke/").install Dir["*"]
+    system "go", "build", "-ldflags",
+           "-w -X main.VERSION=#{version}",
+           "-o", "#{bin}/rke",
+           "-v", "github.com/rancher/rke/"
+  end
+
+  test do
+    system bin/"rke", "--version"
```
Comment (fxcoudert) [inline Formula/rke.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/rke.rb (hunk 1 above)
Comment (johnsockwell) [reply Formula/rke.rb]: No problem.

File: Formula/rke.rb (hunk 2 above)
Comment (johnsockwell) [reply Formula/rke.rb]: I updated the test to generate a template cluster.yml configuration file and assert the file was created.

**Comment by fxcoudert** [comment]
Please update to 0.1.10. Other than that, it looks good to go!

**Comment by johnsockwell** [comment]
Anything else I can do to help move this along?

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @johnsockwell for your contribution to Homebrew!

**Pull request closed without merging**